pom.xml

```diff
@@ -32,73 +33,58 @@
     <description>
         A Java SDK to manipulate EIDAS and PSD2 certificate
     </description>
+    <organization>
+        <name>ForgeRock</name>
+        <url>http://www.forgerock.org</url>
+    </organization>
     <url>https://github.com/OpenBankingToolkit/eidas-psd2-sdk/</url>
 
-    <parent>
-        <groupId>com.forgerock.openbanking</groupId>
-        <artifactId>forgerock-openbanking-starter-parent</artifactId>
-        <version>1.0.83</version>
-        <relativePath /> <!-- lookup parent from repository -->
-    </parent>
-
     <properties>
         <bouncycastle.version>1.62</bouncycastle.version>
+        <bcpkix.jdk15on.version>1.68</bcpkix.jdk15on.version>
+        <jaxb.javax.version>2.2</jaxb.javax.version>
         <java.version>11</java.version>
         <maven.compiler.source>1.8</maven.compiler.source>
         <maven.compiler.target>1.8</maven.compiler.target>
         <version.javadoc.plugin>3.0.1</version.javadoc.plugin>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <jwt.nimbus.version>7.9</jwt.nimbus.version>
+        <junit.version>4.13.1</junit.version>
+        <json.smart.version>2.3</json.smart.version>
+        <license-maven-plugin.version>3.0</license-maven-plugin.version>
     </properties>
 
     <dependencyManagement>
     </dependencyManagement>
 
     <dependencies>
-        <dependency>
-            <groupId>junit</groupId>
-            <artifactId>junit</artifactId>
-            <scope>test</scope>
-            <version>4.12</version>
-        </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcprov-jdk15on</artifactId>
+            <version>${bouncycastle.version}</version>
         </dependency>
         <dependency>
             <groupId>org.bouncycastle</groupId>
             <artifactId>bcpkix-jdk15on</artifactId>
-            <version>1.55</version>
+            <version>${bcpkix.jdk15on.version}</version>
```

Comment:
Why was this change necessary?

Reply:
Isn't necessary but it's more clean to have the versions in the properties.

Reply:
I mean the version change. I remember I had to be very specific with the version or it didn't work! Hopefully the tests will be sufficient to determine if the code still works with these newer versions of the bouncy castle libs!!